cognitive/src/main/scala/com/microsoft/ml/spark/cognitive/TextAnalyticsSDK.scala

```diff
@@ -194,19 +207,25 @@ class TextSentimentV4(override val uid: String)
   extends TextAnalyticsSDKBase[SentimentScoredDocumentV4]() {
   logClass()
 
-  def this() = this(Identifiable.randomUID("KeyphraseExtractionV4"))
+  def this() = this(Identifiable.randomUID("TextSentimentV4"))
 
   override val responseBinding: SparkBindings[TAResponseV4[SentimentScoredDocumentV4]] = SentimentResponseV4
 
   override def invokeTextAnalytics(client: TextAnalyticsClient,
-                                   options: Option[TextAnalyticsRequestOptions],
                                    input: Seq[String],
                                    lang: Seq[String]): TAResponseV4[SentimentScoredDocumentV4] = {
+
     val documents = (input, lang, lang.indices).zipped.map { (doc, lang, i) =>
       new TextDocumentInput(i.toString, doc).setLanguage(lang)
     }.asJava
 
-    val response = client.analyzeSentimentBatchWithResponse(documents, options.orNull, Context.NONE).getValue
+    val options = new AnalyzeSentimentOptions()
```

Comment:
if this pattern is good, i ll update other functions as well

Reply:
LGTM for first pass